tools.js

```diff
@@ -0,0 +1,45 @@
+const normalizeVersion = require('./normalize-version');
+const { execSync } = require("child_process");
+
+module.exports = {
+  node: {
+    getVersion: "node --version",
+    getInstallInstructions(v) {
+     try {
+      // check for existance of nvm
+      execSync(
+        'nvm',
```

Comment:
I was going to suggest running `which nvm` instead of actually executing nvm, but I realize that wouldn't work on Windows. Not sure if it's worth it, but maybe consider using: https://github.com/mathisonian/command-exists

Reply:
Yeah, no `which` on Windows. I also looked at alternatives for Windows. As it turns out, `command-exists` uses `where` under the hood, which works fine in `cmd`, but not in `powershell`. Since `node` uses `cmd`, that would work. However, the check would basically be the same, since `where` also throws if the requested executable is not found.

`command-exists` is overkill for this here.

Additionally, in my Linux-VM (Ubuntu Trusty from osboxes.org, with nvm installed and working), neither `which` nor `command-exists` correctly find `nvm`. I don't know the reason, however.

Both `nvm` and `nvm-windows` have no side effects when called without parameters.

Therefore, I think the implemented check is fine as it is.